src/core/cgroup.h

```diff
@@ -47,6 +47,14 @@ typedef enum CGroupDevicePolicy {
         _CGROUP_DEVICE_POLICY_INVALID = -1
 } CGroupDevicePolicy;
 
+typedef enum FreezerAction {
+        FREEZER_THAW,
+        FREEZER_FREEZE,
```

Comment:
I know this is close to OCD, but since I think another push will be necessary anyway, please bear with me ;)

"freeze" happens before "thaw", so the order of definition would be better reversed.